banzai_nres/tests/adding_traces_to_images_utils.py

```diff
@@ -0,0 +1,85 @@
+from banzai_nres.tests.utils import gaussian, noisify_image, trim_image, FakeImage
+from banzai_nres.utils import trace_utils
+import numpy as np
+
+
+def fill_image_with_traces(image, trimmed_shape, order_width=1.25, odd_fiber_intensity=1E4, even_fiber_intensity=5E3):
+    """
+    :param image: Banzai image object.
+    :param trimmed_shape: The trimmed shape of the image, tuple (rows, columns)
+    :param order_width: the standard deviation of an unnormalized guassian e.g. sigma
+           from np.exp(-(x - b) ** 2 / (2 * sigma ** 2))
+    :param odd_fiber_intensity: peak intensity of the 'odd' fiber (e.g. all fibers starting at index 1, stepping by 2)
+    :param even_fiber_intensity: peak intensity of the 'even' fiber (e.g. all fibers starting at index 0, stepping by 2)
+    Appends the bright streaks across the detector onto image.data according to image.trace.coefficients.
+    """
+    image.data = np.zeros_like(image.data)
+    even_fiber = np.zeros(trimmed_shape)
+    odd_fiber = np.zeros(trimmed_shape)
+    image.trace.image_width = image.data.shape[1]
+    trace_values_versus_xpixel, num_traces, x = image.trace.get_trace_centers()
+    vgauss = np.vectorize(gaussian)  # prepare guassian for evaluation along a slice centered at each trace point.
+    # these are realistic intensity values according to a 120 sec LSC exposure.
+    for x_pixel in range(even_fiber.shape[1]):
```

Comment:
I suspect we can do this in a simpler way without invoking a for loop over pixels.